docs/sources/next/javascript-api/k6-experimental/browser/keyboard/type.md

```diff
@@ -1,6 +1,20 @@
 ---
 title: 'type(text[, options])'
-description: 'TODO'
+description: 'Dispatches a `keydown`, `keypress` or`input`, and `keyup` events for each character in the text.'
 ---
 
 # type(text[, options])
+
+Dispatches a `keydown`, `keypress` or`input`, and `keyup` events for each character in the text. This is useful for simulating typing text into an input field.
+
+| Parameter     | Type   | Default | Description                                                    |
+| ------------- | ------ | ------- | -------------------------------------------------------------- |
+| text          | string | `''`    | The text to type into the input field.                         |
+| options       | object | `null`  | Optional settings.                                             |
+| options.delay | number | `0`     | Optional. Time to wait before typing the text in milliseconds. |
```

Comment:
Or is this the delay in between each key press?

Reply:
See this: https://github.com/grafana/k6-docs/pull/1592/commits/e6dc7ce3f10f01b8bb2519a0f2fcfd6c5b4e7dbe